scripts/install

```diff
@@ -59,25 +59,26 @@ ynh_app_setting_set --app=$app --key=port --value=$port
 #=================================================
 # CREATE DEDICATED USER
 #=================================================
-ynh_script_progression --message="Configuring system user..."
+ynh_script_progression --message="Configuring system user…"
 
 # Create a system user
-ynh_system_user_create --username=$app
+ynh_system_user_create --username=$app --use_shell
 
 #=================================================
 # NGINX CONFIGURATION
 #=================================================
-ynh_script_progression --message="Configuring NGINX web server..."
+ynh_script_progression --message="Configuring NGINX web server…"
 
 # Create a dedicated NGINX config
+sed -i "s@__PORT__@$port@g" ../conf/nginx.conf
```

Comment:
should not be needed, as `ynh_add_nginx_config` use now `ynh_add_config` https://yunohost.org/fr/packaging_apps_helpers?q=%2Fpackaging_apps_helpers#ynh_add_config

Reply:
Thanks for the info! Done.